src/functionalTest/java/uk/gov/hmcts/reform/blobrouter/envelope/ZipFileHelper.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.blobrouter.envelope;
+
+import com.google.common.io.Resources;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+import static com.google.common.io.Resources.getResource;
+
+public class ZipFileHelper {
```

Comment:
Should utility class be marked as final?

Reply:
done